src/System.Drawing.Common/src/System/Drawing/SystemFonts.cs

```diff
@@ -21,31 +19,12 @@ public static Font CaptionFont
                 var data = new NativeMethods.NONCLIENTMETRICS();
                 bool result = UnsafeNativeMethods.SystemParametersInfo(NativeMethods.SPI_GETNONCLIENTMETRICS, data.cbSize, data, 0);
 
-                if (result && data.lfCaptionFont != null)
```

Comment:
Why did you remove the != null check?  What guarantee is there that it's non-null even if result is true?